packages/components/organisms/f-checkout/src/services/tests/mapper.test.js

```diff
@@ -0,0 +1,95 @@
+import mapCheckoutRequest from '../mapper';
+
+const defaultParams = {
+    address: {},
+    customer: {},
+    isCheckoutMethodDelivery: true,
+    time: {},
+    userNote: ''
+};
+
+describe('checkout mapper', () => {
+    it('should map customer details correctly', () => {
+        // Arrange
+        const customer = {
+            firstName: 'Test',
+            lastName: 'Tester',
+            mobileNumber: '07890123456'
+        };
+
+        // Act
+        const requestBody = mapCheckoutRequest({
+            ...defaultParams,
+            customer
+        });
+
+        // Assert
+        expect(requestBody.customer.firstName).toBe(customer.firstName);
+        expect(requestBody.customer.lastName).toBe(customer.lastName);
+        expect(requestBody.customer.phoneNumber).toBe(customer.mobileNumber);
+        expect(requestBody.customer.dateOfBirth).toBe(null);
+    });
+
+    it('should map address correctly', () => {
+        // Arrange
+        const address = {
+            line1: '1 Bristol Road',
+            line2: 'Flat 1',
+            city: 'Bristol',
+            postcode: 'BS1 1AA'
+        };
+
+        // Act
+        const requestBody = mapCheckoutRequest({
+            ...defaultParams,
+            address
+        });
+
+        // Assert
+        expect(requestBody.fulfilment.location.address.postalCode).toBe(address.postcode);
+        expect(requestBody.fulfilment.location.address.lines).toStrictEqual([
+            address.line1,
+            address.line2,
+            '',
+            address.city,
+            ''
+        ]);
+    });
+
+    it('should not map address if checkout method is not delivery', () => {
```

Comment:
It would be nice to wrap some of these in describes. It will help describe the scenario better. For example:

describe('when the checkout method is not delivery') => it('should not map the address')

helps short IT descriptions too.